src/pages/Login/Signup.styles.scss

```diff
@@ -0,0 +1,364 @@
+@import url('https://fonts.googleapis.com/css?family=Biryani&display=swap');
```

Comment:
SCSS needs to be enclosed. Rename it to Signup.module.scss. [documentation](https://create-react-app.dev/docs/adding-a-css-modules-stylesheet)